client/src/components/Onboarding/OnboardingAdmin.js

```diff
@@ -204,13 +195,8 @@ const OnboardingAdminListUser = ({ user, viewLogCallback }) => {
                   : null
               }
             />
-          </div>
-        ))}
       </td>
-      <td>
-        {user.steps.map(step => (
-          <div
-            key={uuidv4()}
+          <td
```

Comment:
(commenting here so it can be resolved)
This may be out of scope for this PR, but when the status is `Processing`, it appears truncated. Is this intentional? Do you see this as well? Full screen, chrome.
<img width="246" alt="Screen Shot 2021-03-18 at 4 55 34 PM" src="https://user-images.githubusercontent.com/20326896/111703054-1d265000-880b-11eb-8bc0-0a789f8b5bfe.png">

Reply:
I do not. When I try to replicate (hack) this state (because normally the processing appears too quickly for me to investigate), I get a loading spinner _beneath_ the pill, not to the side of it. I prefer what you see. Please, send me a link to the ticket (you privately proposed you'd make) after you've made it.

Reply:
Ticket: https://jira.tacc.utexas.edu/browse/FP-972

Reply:
I missed a question. Answer: Not intentional (but not disliked either, personally). I'll suggest Design feedback before changing this.

Reply:
I reproduced the issue for User view (but not Onboarding view). Design input is unnecessary. In https://github.com/TACC/Core-Portal/pull/370, I've made functional Joon Yee's inactive fix.